### ISSUE TITLE
Twig_Autoloader removed in favor of Composer

### ISSUE THREAD

**Issue by curtisghanson** [timber/timber#1370]
[Timber/Timber.php](https://github.com/timber/timber/blob/master/lib/Timber.php#L75) checks for the Twig_AutoLoader class. As of Twig 2.0.0, the class has been removed, see [Twig/CHANGELOG](https://github.com/twigphp/Twig/blob/563f6d0de423c9b798d8177a7c74a4ff76939bd9/CHANGELOG#L29). This affects all installations of Timber, see [Timber/composer.json](https://github.com/timber/timber/blob/master/composer.json#L32).

**Comment by yantei** [comment]
quite serious bug, breaks site after composer update command

**Comment by Twansparant** [comment]
Jup same here. Getting fatal error:
```
Fatal error: You have not run "composer install" to download required dependencies for Timber, you can read more on https://github.com/timber/timber#installation in /srv/www/example.dev/current/vendor/timber/timber/lib/Timber.php on line 76
```

**Comment by exophunk** [comment]
Confirming that Twig 2.2.0, Timber 1.3.0 and WP 4.7.3 do not work anymore. Same error as @Twansparant.

Any way to fix this while staying on 1.3.0 ?

**Comment by jarednova** [comment]
Thanks so much for this report, I'll take a look at that ASAP. @curtisghanson — if you comment out lines 75-77, does everything work as expected?

**Comment by curtisghanson** [comment]
Yes. That's how I got around it. I'd submit a pull request but I'm new to this project so I don't want to try and make changes before I know what I'm doing.

**Comment by jarednova** [comment]
@curtisghanson @yantei @exophunk @Twansparant: thanks so much for your flagging and reporting on this issue. I was very confident of the Twig 2.0 and PHP 7 compatibility based on earlier Travis tests passing — but I failed to read the logs which showed that PHP 7 was actually using Twig 1.x despite the composer spec.

Regardless, I've created #1379 that now shows everything working. I'm going to merge this in and release as `1.3.0-beta2` — I'd love any feedback you're able to give about whether it resolves the issues you saw (or creates any new ones). 

Thanks again!

**Comment by jarednova** [comment]
@curtisghanson @yantei @exophunk @Twansparant: I've merged #1379 which resolves the issue here. Give it a whirl and let me know if it works, as highlighted above my next step is to update a beta release and see if anything else shakes out before making 1.3.0 official
